concordium-consensus/src/Concordium/GlobalState/TreeState.hs

```diff
@@ -30,7 +30,7 @@ import Concordium.GlobalState.Classes
 import Concordium.GlobalState.Finalization
 import Concordium.GlobalState.Parameters
 import Concordium.GlobalState.Statistics
-import Concordium.GlobalState.TransactionTable
+import Concordium.GlobalState.TransactionTable hiding (numberOfNonFinalizedTransactions)
```

Comment:
It is a minor thing, but could we avoid this problem by choosing a different name for the function in the transaction table. This `hiding` thing is ugly.